cdap-security/src/main/java/io/cdap/cdap/security/auth/AccessTokenIdentityExtractor.java

```diff
@@ -30,7 +31,7 @@
  * The AccessTokenIdentityExtractor attempts to extract an {@link AccessToken} included in the Authorization header,
  * validate it, then transform it into a {@link UserIdentityPair}.
  */
-public class AccessTokenIdentityExtractor implements UserIdentityExtractor {
+public class AccessTokenIdentityExtractor extends AbstractIdleService implements UserIdentityExtractor {
```

Comment:
Where is this getting start/stop?

Reply:
Fixed, discussed offline that we should propagating the Service interface.